contracts/discovery/GNS.sol

```diff
@@ -286,6 +311,8 @@ contract GNS is GNSV1Storage, GraphUpgradeable, IGNS, Multicall {
             "GNS: Cannot publish a new version with the same subgraph deployment ID"
         );
 
+        nameSignals[_graphAccount][_subgraphNumber].reserveRatio = reserveRatio;
```

Comment:
Maybe we could move this to the `_upgradeNameSignal` function, right before `namePool.subgraphDeploymentID = _newSubgraphDeploymentID`. What do you think?

Reply:
Yeah good idea! We are using `namePool` in that function already :)